charts/flyte-core/templates/admin/configmap.yaml

```diff
@@ -45,9 +45,7 @@ data:
   notifications.yaml: |
     notifications:
       type: {{ .Values.workflow_notifications.config.notifications.type }}
-      {{- with .Values.workflow_notifications.config.notifications.region }}
-      region: {{- tpl (.) $ }}
-      {{- end }}
+      region: {{ tpl .Values.workflow_notifications.config.notifications.region $ }}
```

Comment:
Just to confirm, region is a required field, right?

Reply:
Yes. These are currently being used and are required until we fully remove the usage of this deprecated field and use it from AWSConfig in  flyteadmin.

https://github.com/flyteorg/flyteadmin/blob/master/pkg/async/notifications/factory.go#L58
https://github.com/flyteorg/flyteadmin/blob/master/pkg/async/notifications/factory.go#L92
https://github.com/flyteorg/flyteadmin/blob/master/pkg/async/notifications/factory.go#L143